src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -685,6 +686,8 @@ public static String getInstructorHoverMessageForFeedbackSession(FeedbackSession
     
     /**
      * Returns the links of actions available for a specific session
+     * @param hasActions
+     *         Indicates whether or not there are possible actions in the first place
```

Comment:
See the file `InstructorHomePageData.java` for instance. Previously, if they catch an `EntityDoesNotExistException`, the actions will be an empty string. This is the equivalent of it.
Although really I haven't managed to think of a case where the actions will be an empty string.